.git-blame-ignore-revs

```diff
@@ -0,0 +1,8 @@
+# This contains a list of git commit hashes that should be ignored when viewing the git blame on github.
+# It can also be passed to `--ignore-revs-file` if using git locally.
+# https://git-scm.com/docs/git-blame#Documentation/git-blame.txt---ignore-revltrevgt
+# A typical use of these would be if a large formatting change was applied across the whole codebase but otherwise
+# no runtime changes were made. Each entry should be preceded with a comment explaining the change.
+# https://black.readthedocs.io/en/stable/guides/introducing_black_to_your_project.html#avoiding-ruining-git-blame
+# Apply noqa markers for all TCH001 violations
+<REPLACE_WITH_FULL_COMMIT_HASH>
```

Comment:
Pending any suggestions.
This line is the only thing that will change in this PR before merging.